src/opera/pge/dswx_s1/schema/algorithm_parameters_s1_schema.yaml

```diff
@@ -8,8 +8,13 @@ runconfig:
 
         # Polarizations to be used for DSWx-SAR
         # [polarizations] for list of specific frequency(s) e.g. [VV, VH] or [VV]
+        # 'dual-pol', 'co-pol', 'cross-pol' will search the polarizations Input GeoTiff files have.
```

Comment:
Just curious why "Input" is capitalized in the comment?

Reply:
These comments are lifted from the example SAS runconfig, so probably just a lingering typo.